permission_handler_android/lib/src/android_object_mirrors/activity.dart

```diff
@@ -81,17 +81,17 @@ class Activity extends JavaObject {
   /// and
   /// https://developer.android.com/reference/androidx/core/app/ActivityCompat.OnRequestPermissionsResultCallback.
   Future<PermissionRequestResult> requestPermissions(
-    List<String> permissions,
+    List<String> permissions, [
     int? requestCode,
-  ) {
+  ]) {
```

Comment:
Can we make the `requestCode` an optional named parameter, instead of an optional positional parameter?

Named parameters are more developer friendly as they show up better in the IDE and also allows for a bit more flexibility in case we need to add additional parameters.